pkg/agent/openflow/client.go

```diff
@@ -97,14 +96,13 @@ type Client interface {
 	// InstallEndpointFlows.
 	UninstallEndpointFlows(protocol binding.Protocol, endpoints []proxy.Endpoint) error
 
-	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer and ClusterIP. It installs the
-	// flow that uses the group/bucket to do service LB. If the affinityTimeout is not zero, it also installs the flow
-	// which has a learn action to maintain the LB decision. The group with the groupID must be installed before,
-	// otherwise the installation will fail.
-	// nodeLocalExternal represents if the externalTrafficPolicy is Local or not. This field is meaningful only when
-	// the svcType is NodePort or LoadBalancer.
-	// nested represents if the Service has the Endpoints which is other Service's ClusterIP.
-	InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16, nodeLocalExternal bool, svcType v1.ServiceType, nested bool) error
+	// InstallServiceFlows installs flows for accessing Service NodePort, LoadBalancer, ExternalIP and ClusterIP. It
+	// installs the flow that uses the group/bucket to do Service LB. If the affinityTimeout is not zero, it also
+	// installs the flow which has a learn action to maintain the LB decision. The group with the groupID must be
+	// installed before, otherwise the installation will fail.
+	// externallyAccessible indicates that whether the Service is externally accessible, like NodePort, LoadBalancer and ExternalIP.
+	// nested indicates that whether the Service has the Endpoints which is other Service's ClusterIP.
```

Comment:
"whether the Service's Endpoints are other Services' ClusterIPs"?

Reply:
Updated to "// nested indicates whether the Service IP serves as an Endpoint for another Service."

Reply:
I think "nested" means the Service' Endpoints are ClusterIPs of other Services? No?

Or how about "nested indicates the Service are backed by Endpoints of other Services"?

Reply:
Thanks, much better.